library/std/src/sys/windows/path/absolute/tests.rs

```diff
@@ -0,0 +1,149 @@
+// Original implementation taken from crate "normpath".
+// Copyright 2020 "dylni" (https://github.com/dylni)
+
+// Most test cases were taken from:
+// https://googleprojectzero.blogspot.com/2016/02/the-definitive-guide-on-win32-to-nt.html
+
+// These tests assume that the current drive is "C:".
+
+use crate::env;
+use crate::io;
+use crate::path::Path;
+
+fn assert_eq(left: &Path, right: &Path) {
+    assert_eq!(left.as_os_str().bytes(), right.as_os_str().bytes());
```

Comment:
You shouldn't need `.bytes()` here. Comparing `OsStr`s is fine and provides nicer panic messages should it fail.

Reply:
This is strange. I don't know why I added `.bytes()`.